src/components/Documentation/Articles.js

```diff
@@ -126,12 +140,27 @@ const Articles = ({
     });
   };
 
+  // `articles` is keyed by directory or file name.
+  // Directories have a leading `/`, "index" here represents the filename
+  const indexArticle = articles.index;
+
   return (
-    <li key={id}>
+    <El>
+      {/* eslint-disable-next-line */}
       {isNested ? (
-        <NestedArticleTopicExpander onClick={() => topicToggleHandler()}>
-          {title}
-        </NestedArticleTopicExpander>
+        indexArticle ? (
```

Comment:
Sorry, this is gross, but I didn't want to pull it off into a variable. Can revisit if you'd rather not see a nested ternary (which I get)